src/TrueLayer/PaymentsProviders/Model/SearchPaymentsProvidersRequest.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using TrueLayer.Models;
+
+namespace TrueLayer.PaymentsProviders.Model;
+
+public record SearchPaymentsProvidersRequest(AuthorizationFlow AuthorizationFlow,
+    List<string>? Countries = null,
+    List<string>? Currencies = null,
+    List<string>? ReleaseChannels = null,
+    List<string>? CustomerSegments = null,
+    Capabilities? Capabilities = null);
```

Comment:
I've noticed that we usually use `string` types and provide predefined values. Eg see the Currencies class. 
Shall we switch to enum at some point, or we prefer using strings for some reasons? I see pros and cons. (The Java lib uses enums)

Reply:
We can always revisit everything later if necessary. Enums are good though. Not sure why strings were used  initially.